test/tests/xe/xe_network_instance_ospfv2_global.yml

```diff
@@ -635,3 +635,117 @@
+            openconfig-network-instance:network-instances:
+              openconfig-network-instance:network-instance:
+                - openconfig-network-instance:name: 'default'
+                  openconfig-network-instance:config:
+                    openconfig-network-instance:name: 'default'
+                    openconfig-network-instance:type: 'DEFAULT_INSTANCE'
+                    openconfig-network-instance:enabled: true
+                  openconfig-network-instance:protocols:
+                    openconfig-network-instance:protocol:
+                      - openconfig-network-instance:identifier: 'OSPF'
+                        openconfig-network-instance:name: '1'
+                        openconfig-network-instance:config:
+                          openconfig-network-instance:enabled: True
+                          openconfig-network-instance:identifier: 'OSPF'
+                          openconfig-network-instance:name: '1'
+                        openconfig-network-instance:ospfv2:
+                          openconfig-network-instance:areas:
+                            openconfig-network-instance:area:
+                            - openconfig-network-instance:identifier: 3
+                              openconfig-network-instance:config:
+                                openconfig-network-instance:identifier: 3
+                              openconfig-network-instance:interfaces:
+                                openconfig-network-instance:interface:
+                                - openconfig-network-instance:id: 'Loopback10'
+                                  openconfig-network-instance:config:
+                                    openconfig-network-instance:id: 'Loopback10'
+                                    openconfig-ospfv2-ext:authentications:
+                                      openconfig-ospfv2-ext:authentication:
+                                        - openconfig-ospfv2-ext:key-id: 2 
+                                          openconfig-ospfv2-ext:config:
+                                            openconfig-ospfv2-ext:key-id: 2
+                                            openconfig-ospfv2-ext:enabled: true
+                                            openconfig-ospfv2-ext:md5-key: 'pass1'
+                                        - openconfig-ospfv2-ext:key-id: 3 
+                                          openconfig-ospfv2-ext:config:
+                                            openconfig-ospfv2-ext:key-id: 3
+                                            openconfig-ospfv2-ext:enabled: true
+                                            openconfig-ospfv2-ext:md5-key: 'pass2'
+                                          
+        api_method: PUT
+        api_ignore_errors: false
+        rollback: true
+        assertion_ignore_errors: false
+        assertions:
+          - "'+interface Loopback10:' in changes"
+          - "'+ ip ospf authentication:' in changes"
```

Comment:
should be configured for:
          - "'+ ip ospf authentication message-digest:' in changes"